app/models/meter.rb

```diff
@@ -103,7 +103,7 @@ def school_name
   end
 
   def admin_meter_status_label
-    admin_meter_status&.label || school&.school_group&.admin_meter_status&.label || ''
+    admin_meter_status&.label || school&.school_group&.send(:"admin_meter_status_#{fuel_type}")&.label || ''
```

Comment:
Valid meter types are: 

```
enum meter_type: [:electricity, :gas, :solar_pv, :exported_solar_pv]
```

I think the exported solar pv meters should have same default type as solar pv. Can you revise that?